### DISCUSSION TITLE
Flashing when loading 3dtile

### DISCUSSION THREAD

**Discussion by chenkhcn** [visgl/deck.gl#9305] [Q&A]
When using MapboxOverlay and Tile3DLayer to render 3dtiles with Mapbox, there may be flickering issues. How can I handle this

**Comment by eoudejans** [comment]
Is the 3d tileset you are mentioning open for other people to use? Flickering can happen by a multitude of causes. Does the flickering stop when you turn off depth testing?:
```
parameters: {depthTest: false}
```

**Reply by chenkhcn** [reply]
https://gis.easyscdp.com/assets/zs_c_1115/11m13_3dtile.json
Turning off depth detection will also blink

**Reply by eoudejans** [reply]
Do you have other layers besides the 3D tile layer active?

**Reply by chenkhcn** [reply]
no

**Reply by eoudejans** [reply]
Does the flickering fluctuate by changing:
- zoom?
- viewing angle?
